cmd/reset/reset-consumer-group-offset.go

```diff
@@ -0,0 +1,28 @@
+package reset
+
+import (
+	"github.com/deviceinsight/kafkactl/operations/consumergroupoffsets"
+	"github.com/spf13/cobra"
+)
+
+var offsetFlags consumergroupoffsets.ResetConsumerGroupOffsetFlags
+
+var cmdResetOffset = &cobra.Command{
+	Use:     "consumer-group-offset GROUP",
+	Aliases: []string{"cgo", "offset"},
+	Short:   "reset a consumer group offset",
+	Args:    cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		(&consumergroupoffsets.ConsumerGroupOffsetOperation{}).ResetConsumerGroupOffset(offsetFlags, args[0])
+	},
+}
+
+func init() {
+	cmdResetOffset.Flags().BoolVarP(&offsetFlags.OldestOffset, "oldest", "", false, "set the offset to oldest offset (for all partitions or the specified partition)")
+	cmdResetOffset.Flags().BoolVarP(&offsetFlags.NewestOffset, "newest", "", false, "set the offset to newest offset (for all partitions or the specified partition)")
+	cmdResetOffset.Flags().Int64VarP(&offsetFlags.Offset, "offset", "", -1, "set initial offset for a partition")
+	cmdResetOffset.Flags().Int32VarP(&offsetFlags.Partition, "partition", "p", -1, "partition to apply the offset")
```

Comment:
The help for `kafkactl reset consumer-group-offset` looks as follows for this parameter:
```
  -p, --partition int32   partition to apply the offset (default -1)
```

As far as I understand `-1` stands for "all partitions"? IMO this is not very intuitive from a usage perspective. It should at least be mentioned in the help text.